content/soarian/r4/base/individuals/practitioner.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Practitioner | R4 API
+---
+
+# Practitioner
+
+* TOC
+{:toc}
+
+## Overview
+
+The Practitioner resource provides information about a person formally involved in the care of a patient on behalf of a health care facility. 
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
+The search results include the following fields if they contain values:
+
+*	[Practitioner id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+*	[Identifiers/Aliases such as NPI](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.identifier){:target="_blank"}
+*	[Active (true/false)](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.active){:target="_blank"}
+*	[Name](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.name){:target="_blank"}
+
+## Search
+
+Search for Practitioner resources that meet specified query parameters:
+
+	GET /Practitioner?:parameters
+
+_Implementation Notes_
+
+*	A `422 Unprocessable Entity` response status code and OperationOutcome are returned when > 499 Practitioners qualify for the search criteria. Fast Healthcare Interoperability Resources (FHIR) applications may prompt the user for more specific search criteria to avoid this possibility.
```

Comment:
Please add a registered trademark symbol after "Fast Healthcare Interoperability Resources" and "(FHIR)".